moabb/datasets/base.py

```diff
@@ -7,17 +7,63 @@
 class BaseDataset(metaclass=abc.ABCMeta):
     """Base dataset"""
 
-    def __init__(self, subjects, sessions_per_subject, events, code, interval,
-                 paradigm, doi=None):
+    def __init__(self, subjects, sessions_per_subject, events,
+                 code, interval, paradigm, task_interval=None, doi=None):
+        """
+        Parameters required for all datasets
 
+        parameters
+        ----------
+        subjects: List of int
+            List of subject number # TODO: make identifiers more general
+        
+        sessions_per_subject: int
+            Number of sessions per subject
+
+        events: dict of string: int
+            String codes for events matched with labels in the stim channel. Currently imagery codes codes can include:
```

Comment:
we also have elbow and other stuff.

I'm wondering if we should start using MNE hierarchical event definition.

for example, you can define an event as `hand/left` and `hand/right`, which allow to select all `hand` event after epoching by doing `Epochs['hand']`. But this is another discussion

Reply:
Well we lose nothing by adding it so why not, I'll go through and change all left_hand to hand/left etc --although there is one more level to worry about, of imagined vs actual

Reply:
you can do imagined/hand/left, etc

Reply:
but we can skip this for now, and see how we can deal with that later.